client/i18n/locales/english/intro.json

```
@@ -2068,34 +2230,61 @@
           "In this workshop, you'll build a city skyline. You'll practice how to configure CSS variables so you can reuse them whenever you want."
         ]
       },
-      "vlov": { "title": "109", "intro": [] },
+      "vlov": {
+        "title": "109",
+        "intro": []
+      },
       "lab-availability-table": {
         "title": "Build an Availability Table",
         "intro": ["For this lab, you will create an availability table."]
       },
-      "sget": { "title": "111", "intro": [] },
+      "sget": {
+        "title": "111",
+        "intro": []
+      },
       "quiz-css-variables": {
         "title": "CSS Variables Quiz",
         "intro": [
           "Test what you've learned in this quiz of how using variables in CSS."
         ]
       },
-      "agbl": { "title": "113", "intro": [] },
-      "ogko": { "title": "114", "intro": [] },
-      "harp": { "title": "115", "intro": [] },
-      "nsiq": { "title": "116", "intro": [] },
+      "agbl": {
+        "title": "113",
+        "intro": []
+      },
+      "ogko": {
+        "title": "114",
+        "intro": []
+      },
+      "harp": {
+        "title": "115",
+        "intro": []
+      },
+      "nsiq": {
+        "title": "116",
+        "intro": []
+      },
       "lab-product-landing-page": {
         "title": "Build a Product Landing Page",
         "intro": [
           "For this project, you will build a product landing page to market a product of your choice."
         ]
       },
-      "fdam": { "title": "118", "intro": [] },
+      "fdam": {
+        "title": "118",
+        "intro": []
+      },
       "quiz-css-grid": {
         "title": "CSS Grid Quiz",
         "intro": ["Test what you've learned in this quiz of using grid in CSS."]
       },
-      "dorc": { "title": "120", "intro": [] },
+      "workshop-ferris-wheel": {
+        "title": "Draw an Animated Ferris Wheel",
```

This sounds more natural to me:

```suggestion
        "title": "Build an Animated Ferris Wheel",
```